src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockSearchDAO.scala

```diff
@@ -19,43 +20,43 @@ class MockSearchDAO extends SearchDAO {
   override def createIndex = Unit
   override def deleteIndex = Unit
 
-  var indexDocumentInvoked = false
-  var deleteDocumentInvoked = false
-  var findDocumentsInvoked = false
-  var autocompleteInvoked = false
-  var populateSuggestInvoked = false
+  var indexDocumentInvoked = new AtomicBoolean(false)
+  var deleteDocumentInvoked = new AtomicBoolean(false)
+  var findDocumentsInvoked = new AtomicBoolean(false)
+  var autocompleteInvoked = new AtomicBoolean(false)
+  var populateSuggestInvoked = new AtomicBoolean(false)
```

Comment:
this change was my first guess at the problem and even though it was not the fix, it is still technically correct because these things are potentially accessed from multiple threads